clpy/core/core.pyx

```diff
@@ -3527,30 +3524,34 @@ cpdef ndarray matmul(ndarray a, ndarray b, ndarray out=None):
     for i in la:
         batchCount *= i
 
-    ap = _mat_ptrs(a)
-    bp = _mat_ptrs(b)
-    outp = _mat_ptrs(out_view)
+    # Create Numpy ndarrays which contain
+    # offsets in elements of batched sub-arrays.
+    ap = _mat_offsets_in_elements(a)
```

Comment:
I think `ap` means "a's pointer" thus this name is not good on ClPy. How about `offsets_a`, which is the same name of arg of `gemm_batched`?

Reply:
That's right. I've unintendedly left these names. Fixed.